readme.md

```diff
@@ -719,7 +719,6 @@
 | [Soft Components](https://soft-components-docs.web.app/)| A set of framework-agnostic web components based on neumorphic design. |
 | [Fast](https://www.fast.design/)| An interface system that can be used with modern Web Frameworks such as React, Vue and Angular. |
 | [LottieFiles ](https://lottiefiles.com/)| Interactive animations in many formats like json,gif and mp4, libraries and plugins for Web & Mobile . |
-| [Tailwindow ](https://component.tailwindow.com/)| Components created using Tailwind CSS utilities class, so it can be customized to make themes easily. |
```

Comment:
Looks like the page is live

Reply:
All I'm getting is this placeholder page with (spam) links. Are you getting a different page?

![https://component.tailwindow.com](https://i.ibb.co/KrpcJ9Q/Screen-Shot-2021-10-04-at-21-34-24.png)

Reply:
![image](https://user-images.githubusercontent.com/3925474/135996873-a6ac5aa3-332d-4940-9716-1ea913a27689.png)


Looks like it's not 100% mobile friendly.